src/g_game.c

```diff
@@ -872,6 +897,46 @@ static boolean G_StrictModeSkipEvent(event_t *ev)
   return false;
 }
 
+static void CheckLocalView(void)
+{
+  const player_t *player = &players[consoleplayer];
+
+  localview.active = (
```

Comment:
Erm, wow, that's quite a lot to check. I wonder if there's a solution where we just set `localview.active = 0` on demand and keep it enabled by default?

Reply:
This is definitely an area to simplify, if possible. In general, all of the various "interpolation" checks seem like an area that could be improved throughout the entire project.